Makefile

```diff
@@ -634,15 +634,19 @@ promote-images: $(KPROMO) ## Promote images.
 
 ##@ Testing:
 .PHONY: test
-test: generate lint go-test ## Run "generate", "lint" and "go-tests" rules.
+test: generate lint go-test-race ## Run "generate", "lint" and "go-test-race" rules.
+
+.PHONY: go-test-race
+go-test-race: TEST_ARGS+= -race
```

Comment:
This is a [target-specific variable value](https://www.gnu.org/software/make/manual/html_node/Target_002dspecific.html), which seems like the right way to reuse the `go-test` target while adding arguments to `$(TEST_ARGS}`.